extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/FastBootHibernatePersistenceProvider.java

```diff
@@ -404,12 +404,15 @@ private static void injectRuntimeConfiguration(HibernateOrmRuntimeConfigPersiste
             Builder runtimeSettingsBuilder) {
         // Database
         runtimeSettingsBuilder.put(AvailableSettings.JAKARTA_HBM2DDL_DATABASE_ACTION,
-                persistenceUnitConfig.database().generation().generation());
+                persistenceUnitConfig.database().generation().generation()
+                        .orElse(persistenceUnitConfig.schemaManagement().strategy()));
```

Comment:
So, if the user applies both, then the deprecated option will "win"? 
maybe we should check how the two match (or new one is none) and fail ? (if so, the same would apply to other options in this config)

Reply:
Yeah, we sure could. But I think it's going to be a bit annoying as you can't really know if the new config is set or not (if it's set to the default) if you don't make it optional.

And it makes dropping the old stuff a bit more painful.

But if you feel like it, feel free to build on it once this is in.